deploy/setup/scripts/launch.sh

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# --- Expected Variables ---
+
+export LOCALHOST
+export DEBUG
+export LOG_PATH
+export DEPLOY_DIR
+export SCRIPTS_DIR
+export SETUP_DIR
+export SKIP_DEPENDENCY_INSTALLS
+export SKIP_LAUNCH
+
+# --- New Variables ---
+
+# Internal re-entry flag used when ran detached
+LAUNCH_ONLY=$([ "${1-}" = "--launch-only" ] && echo true || echo false)
```

Comment:
Is the comment still up to date? This does not look like a re-entry flag, does it?

Reply:
I see now. Interesting flow. Not obvious on the first sight, but makes sense. Did you consider splitting this into two script files?

Reply:
I will consider it further.